### PR TITLE
Image standards

### PR DESCRIPTION
**Pull request by DennisTsiang**
Fixes style guide images and links. Also adds some image guidelines to the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,3 +58,15 @@ Adding a line with emphasis directly under an image will style it as a caption:
 ![](image_url)
 *Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod.*
 ```
+
+## Image Guidelines
+
+All images, excluding external links, should have a retina version and a
+non-retina version stored in the same repository, with the suffix @2x
+appended onto the filename (E.g macbook.jpg and macbook@2x.jpg). The
+retina version should be twice the size of the non-retina version.
```
Comment (neilvanbeinum) [inline README.md]: We should probably point people at the style guide here so they have a reference for the image dimensions.

**Comment by neilvanbeinum** [comment]
Thanks for updating this @DennisTsiang.

Couple of things that I think we need to update on the style guide:

- The style guide images from placehold.it currently are not showing on the preview URL due to a mixed content error. Updating these to `https` should fix it.
- It's something that should have arguably been done at the time, but I think the style guide would benefit from having some additional elements in it so that devs and e.g. product owners can refer to it for all of the image dimensions that we use on our site (excluding the blog which I don't think has rules at the moment). For instance: `.product-story-large-tile`, `.product-story-small-tile`, `.person-tile`, as well as a quote block with an image and maybe others too. Hopefully mostly a copy-paste job and double checking the correct image dimensions to pass to placehold.it

File: source/style-guide.erb [hunk 2]
```diff
@@ -104,11 +104,49 @@ title: "Style guide"
...
     <div class="style-guide__container-title">
-      Product story related tile
+      Product story quote with image
     </div>
     <div class="style-guide__container-body">
+      <%= partial 'product-stories/quote_caricature', locals: {
+        section: OpenStruct.new({
+          quote: "Consectetur adipisicing elit. Consequuntur sed aliquam vero repellendus impedit qui. Quia labore, quibusdam soluta, corrupti dolor deleniti voluptatum corporis quidem vero, beatae eius voluptatem quod!",
+          source: "Lorem ipsum",
+          image: "http://via.placeholder.com/150x254"
```
Comment (neilvanbeinum) [inline source/style-guide.erb]: We get mixed content warnings for these when viewing the preview site. Can we change them to `https`? (Same for the others below)

File: source/style-guide.erb [hunk 3]
```diff
@@ -123,6 +161,77 @@ title: "Style guide"
...
+      Person tile
+    </div>
+    <div class="style-guide__container-body">
+      <div class="person-tile">
+          <div class="person-tile__content">
+            <div class="anchor">
+              <span id="style-guide-person-tile"></span>
+            </div>
+            <div class="person-tile__image-container">
+              <img class="person-tile__image" src="/assets/images/people/tom-sabin.png" alt="tom sabin" />
```
Comment (neilvanbeinum) [inline source/style-guide.erb]: I wonder if we should also use a placeholder image with dimensions here too. I think it should help when people add new caricatures to use the correct size.

**Review by neilvanbeinum — CHANGES_REQUESTED** [review]

File: source/style-guide.erb (hunk 2 above)
Comment (DennisTsiang) [reply source/style-guide.erb]: Apologies, I don't know how I missed those. I have amended them in the commit.

File: source/style-guide.erb (hunk 3 above)
Comment (DennisTsiang) [reply source/style-guide.erb]: Good idea, will change that to show the placeholder.

**Review by neilvanbeinum — APPROVED** [review]
Looks good! Thanks for the updates. One last thing I wondered was whether we want to group together the "blocks" - so for instance, we could add the whitepaper section above the typography section so its in the same place as the other blocks. I don't feel super strongly about it though.

**Pull request merged**